contents/faq-page/05_misure/001-measure-1-1/000_sectionData.md

```diff
@@ -5,6 +5,8 @@
     sectionId: measure-1-1
     chips:
       - chipsId: measure-1-1
-      - title: Generale
-        id: generale
+      - title: PSN PAC pilota
+        id: psn_pac
+      - title: Altre PAC
+      - id: altre_pac
```

Comment:
Non capisco l'inserimento del chip "Altre PAC" che poi non è applicato a nessuna domanda, così quando seleziono il Chip la risposta è 0 FAQ
Inoltre le domande online per questa sez. sono 37, e qui "ridotte" a 29 ma non vedo files eliminati. Sicuro di avere il tuo Fork aggiornato?

preview: https://padigitale2026-gov-it-site-dvhbk6ux7.vercel.app/supporto/domande-frequenti